frontend/src/features/event-master-management/components/EditEventMasterFormContent.tsx

```
@@ -0,0 +1,47 @@
+"use client";
+
+import { TextField, SelectField, TextareaField } from "@/components/form";
+import { LOCATION_TYPES } from "@/constants/enums";
+import type { EventMasterResponseDto } from "@/types/event-master";
+
+type EditEventMasterFormContentProps = {
+  eventMaster: EventMasterResponseDto;
+};
+
+export const EditEventMasterFormContent =
+  ({}: EditEventMasterFormContentProps) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)